index.html

```diff
@@ -0,0 +1,51 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+
+    <header id="header">
+      <div class="header-container">
+        <div class="logo">
+          <img class="logo-image" id="header-img" src="./images.PNG">
+        </div>
+        <div class="menu-items">
+          <nav id="nav-bar">
+            <ul>
+              <li><a class="nav-link" href="#">Home</a></li>
+              <li><a class="nav-link" href="#">About</a></li>
+              <li><a class="nav-link" href="#">Contact</a></li>
+            </ul>
+          </nav>
+        </div>
+      </div>
+      
+    </header>
+
+    <main>
+      <div class="container" >
+        <form action="https://www.freecodecamp.com/email-submit" id="form">
+          <input type="email" id="email" placeholder="name@youremail.com">
+          <button id="'submit" type="submit">SUBMIT</button>
+        </form>
+      </div>
+      <div class="youtube-ideo" id="video">
+        <iframe width="560" height="315" src="https://www.youtube.com/embed/v0Ch3yWQ-Zc" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+      </div>
+      
+    </main>
+    
+    <script  src="js/index.js"></script>
+    <footer>
+      <span>Instagram</span>
+      <span>facebook</span>
+      <span>Twitter</span>
+      <p>COPYRIGHT </p>
+    </footer>
```

Comment:
I think You can use the Unordered list and make these span elements as list elements.  It helps search engines understand the structure of your website.